### PR TITLE
V1 of the eatery data provider.

### PR DESCRIPTION
**Pull request by Patchrik**

### CODE REVIEW

File: scripts/.Settings.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 export default {
-	graphhopperKey: "",
-	npsKey: "",
-	weatherKey: ""
-}
-
+	graphhopperKey: '',
+	npsKey: '4CCbkNphNpJ3QSNIMKkkLeLl2DrSerPMI1iTBQl1',
```
Comment (brendalong) [inline scripts/.Settings.js]: You should not pass the API key to gitHub

**Review by brendalong — COMMENTED** [review]
Check comments

**Review by j0verton — DISMISSED** [review]
looks good

**Review by mnmccoy95 — APPROVED** [review]
Looks great!

**Review by j0verton — APPROVED** [review]
!

**Pull request merged**